llvm/include/llvm/Analysis/VecFuncs.def

```diff
@@ -21,6 +21,10 @@
 #define TLI_DEFINE_VECFUNC(SCAL, VEC, VF, VABI_PREFIX) {SCAL, VEC, VF, NOMASK, VABI_PREFIX},
```

Comment:
Rather than leave this to chance it would be better to set CONV to std::nullopt explicitly wherever TLI_DEFINE_VECFUNC is defined.  You'll see we do this already for the mask parameter and from what I can see there are only a few relevant definitions.